poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
```

Comment:
I guess this file didnt need updating but doesnt look like anything changed apart from markup 3.01 to 3.02